css/amp.css

```diff
@@ -146,7 +146,12 @@ i-amp-sizer {
   /* Just state. */
 }
 
+
+/** Only show the last placeholder element */
 .-amp-element > [placeholder] {
+  display: none;
+}
+.-amp-element > [placeholder]:last-of-type {
```

Comment:
I found a way to do this but for the **first** match [using ~ selector](http://jsfiddle.net/Vq8PB/216/) 
See: http://stackoverflow.com/a/8539107/646979

However, I doubt this can work somehow for last match. So one way to work this out is for us to adapt the **first** placeholder instead of the last. The problem with this is that in case the double placeholder problem happens, our CSS would choose the dynamic placeholder instead of the user provided one which doesn't really solve the whole problem.